src/main/scala/com/gu/fastly/Lambda.scala

```diff
@@ -23,8 +23,10 @@ class Lambda {
       (event.itemType, event.eventType) match {
         case (ItemType.Content, EventType.Delete) =>
           sendPurgeRequest(event.payloadId)
+        case (ItemType.Content, EventType.Update) =>
+          sendSoftPurgeRequest(event.payloadId)
         case other =>
-          // for now we only send purges for content takedowns, so ignore any other events
+          // for now we only send purges for content, so ignore any other events
```

Comment:
Ah, looks like the thrift file was copied into this project. You can delete it now and add a dependency on: https://github.com/guardian/content-api-models. Then you'll have the up to date file.